src/patches/vs1053b-patches-latm.plg

```diff
@@ -0,0 +1,762 @@
+/* User application code loading tables for VS10xx */
+#if 0
+void LoadUserCode(void) {
+  int i = 0;
+
+  while (i<sizeof(plugin)/sizeof(plugin[0])) {
+    unsigned short addr, n, val;
+    addr = plugin[i++];
+    n = plugin[i++];
+    if (n & 0x8000U) { /* RLE run, replicate n samples */
+      n &= 0x7FFF;
+      val = plugin[i++];
+      while (n--) {
+        WriteVS10xxRegister(addr, val);
+      }
+    } else {           /* Copy run, copy n samples */
+      while (n--) {
+        val = plugin[i++];
+        WriteVS10xxRegister(addr, val);
+      }
+    }
+  }
+}
+#endif
+
+#ifndef SKIP_PLUGIN_VARNAME
+const unsigned short patches-latm[] = { /* Compressed plugin */
```

Comment:
Let's use snake-case or camel-case for a const name.

Reply:
Snake case was a new one for me.

Reply:
I personally prefer Camel Case. 
However, a project has some legacy background. For example the `write_register` is snake as derived from `Edzelf/Esp-radio`. My fault, maybe we could give a new name for a public member. I should catch it while approving PR.

This is not good that a single project uses two styles. But let's say, low level ones are like written in such `_` manner. I know, a little bit like bending facts to reality ;-)

In general I would rather use Camel Case.
I tried to adjust a library API to Arduino style where Camel Case is used rather. 
That's why names like `switchToMp3Mode()`.

Please see examples in here:
https://www.arduino.cc/en/Reference/StyleGuide

```c
if (somethingIsTrue) doSomething;
```

On the other hand this is a const: `const unsigned short patches-latm[]`. Maybe then, as any other `const` (eg. SKIP_PLUGIN_VARNAME a line above) this should be `PATCHES_LATM[]`? We were probably mislead by `.plg` file authors who used smaller case instead of the upper case here.

Reply:
I see that in my own projects all the time. Sometimes it is hard to keep to a style.

Reply:
What do you think, about making them all upper case? That would follow one convention if about constants naming?

Eg. PATCHES, PATCHES_LATM, PATCHES_FLAC_LATM, ... ?